caf_solution/local.maps.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  maps = merge(
+    var.maps,
+    {
+      maps_account  = var.maps_account
```

Comment:
```suggestion
      maps_accounts  = var.maps_accounts
```

Reply:
Hi @arnaudlh, I have just updated. Please help to review it again.